prime-router/metadata/hl7_mapping/receivers/STLTs/CA/CA-receiver-transform.yml

```diff
@@ -224,10 +214,6 @@ elements:
       %resource.value.exists().not() and 
       %resource.name.exists()
 
-  # override xpn-person-name.use to set a default
-  - name: use
```

Comment:
Same here, seems like we should keep this override

Reply:
the element `use` doesn't exists anymore. The default backup is covered in `ca-patient.yml`